### ISSUE TITLE
Global services

### ISSUE THREAD

**Issue by WalterLenuzza** [rackerlabs/rackspace-sdk-plugin#18]
I believe this [line](https://github.com/rackerlabs/rackspace-sdk-plugin/blob/master/rackspace/connection.py#L38) assumes that every endpoint is prefixed by the region but that's not always the case in a Rackspace Cloud account. For example:

* [CDN](https://developer.rackspace.com/docs/cdn/v1/developer-guide/#document-general-api-info/service-access-endpoints) - 
`https://global.cdn.api.rackspacecloud.com`
* [Cloud Monitoring](https://developer.rackspace.com/docs/cloud-monitoring/v1/developer-guide/#document-general-api-info/service-access-endpoint) - 
`https://monitoring.api.rackspacecloud.com`

**Comment by briancurtin** [comment]
Yep, we may either need to set the region to `""` or `None` or something for the time being on global services instead of applying to everything. I haven't dug into this, but we'll have to figure something out. 
However, I'd prefer to wait until we switch out our session handling code upstream in place of the keystoneauth package, which handles all of the service catalog parsing and preferences. That review is almost through and should be released shortly.
